Packs/Linkshadow/Integrations/Linkshadow/Linkshadow.py

```diff
@@ -0,0 +1,209 @@
+    if last_fetch is None:
+        last_fetch = first_fetch_time
+    else:
+        last_fetch = int(last_fetch)
+    latest_created_time = int(last_fetch)
+    incidents = []
+    alerts = client.fetch_anomaly(apiKey=apiKey, api_username=api_username,
+                                  plugin_id=plugin_id, action=action, time_frame=time_frame)
+    alert = {}
+    for alert in alerts:
+
+        if alert == 'data':
+            for dic in alerts['data']:
+                for key in dic.keys():
+                    if key == 'time_seen':
+                        incident_created_time = dic['time_seen']
+                        if last_fetch:
+                            if int(incident_created_time) <= last_fetch:
+                                continue
+                        incident_name = "Linkshadow-entityAnomaly"
+                        formatted_JSON = format_JSON_for_fetch_incidents(dic)
+
+                        incident = {
+                            'name': incident_name,
+                            'occurred': timestamp_to_datestring(incident_created_time),
+                            'rawJSON': json.dumps(formatted_JSON),
+                            'CustomFields': {  # Map specific XSOAR Custom Fields
+                                'sip': formatted_JSON['sip'],
+                                'sourceip': formatted_JSON['sip'],
+                                'destinationip': formatted_JSON['dip'],
+                                'sourceport': formatted_JSON['sport'],
+                                'destinationport': formatted_JSON['dport'],
+                                'macaddress': formatted_JSON['smac'],
+                                'alertid': formatted_JSON['anomaly_id'],
+                                'subcategory': formatted_JSON['category']
+                            }
+                        }
+                        incidents.append(incident)
+                        # Update last run and add incident if the incident is newer than last fetch
+                        # if incident_created_time > latest_created_time:
+                        #     latest_created_time = incident_created_time
+                        # print (max_alerts)
+                        if len(incidents) >= max_alerts:
+                            break
+    next_run = {'last_fetch': latest_created_time}
+    return next_run, incidents
+
+
+def fetch_entity_anomalies(client, args):
+    apiKey = args.get('apiKey'),
+    username = args.get('api_username')
+    plugin_id = args.get('plugin_id')
+    action = args.get('action')
+
+    time_frame = arg_to_number(
+        arg=args.get('time_frame'),
+        arg_name='time_frame',
+        required=False
+    )
+
+    alerts = client.fetch_anomaly(
+        apiKey=apiKey,
+        api_username=username,
+        plugin_id=plugin_id,
+        action=action,
+        time_frame=time_frame
+    )
+
+    results = CommandResults(
+        outputs_prefix='Linkshadow.anomaly',

```

Comment:
This should and does not match the context path defined in the yml file - `Linkshadow.data.FIELD_NAME`.
let's change this (in the yml definition or here) to match the object that gets returned.